### PR TITLE
caffeine: add page

### PR DESCRIPTION
**Pull request by franzqat**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by kbdharun** [comment]
@navarroaxel I think this page should be placed in `common` as it would work in mac too. What do you think?

**Comment by franzqat** [comment]
@kbdharun The reason why I placed it in linux page is because of Issue #8138

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. 

> Also, Thanks for linking the issue, I too agree having separate pages would be better for `caffeine`.

File: pages/linux/caffeine.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Prevent desktop idleness in full-screen mode.
+> More information: <https://manned.org/caffeine>.
+
+- Start caffeine server:
+
+`caffeine`
+
+- Show the help and exit:
+
+`caffeine -h`
```
Comment (navarroaxel) [inline pages/linux/caffeine.md:12]: We prefer long arguments if available
```suggestion
`caffeine --help`
```

File: pages/linux/caffeine.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Show program's version number and exit:
+
+`caffeine -V`
```
Comment (navarroaxel) [inline pages/linux/caffeine.md:16]:
```suggestion
`caffeine --version`
```

File: pages/linux/caffeine.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# caffeine
+
+> Prevent desktop idleness in full-screen mode.
+> More information: <https://manned.org/caffeine>.
+
+- Start caffeine server:
+
+`caffeine`
+
+- Show the help and exit:
```
Comment (navarroaxel) [inline pages/linux/caffeine.md:10]: This is a standard description
```suggestion
- Display help:
```

File: pages/linux/caffeine.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`caffeine -h`
+
+- Show program's version number and exit:
```
Comment (navarroaxel) [inline pages/linux/caffeine.md:14]:
```suggestion
- Display version:
```

File: pages/linux/caffeine.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# caffeine
+
+> Prevent desktop idleness in full-screen mode.
+> More information: <https://manned.org/caffeine>.
+
+- Start caffeine server:
```
Comment (navarroaxel) [inline pages/linux/caffeine.md:6]:
```suggestion
- Start a caffeine server:
```

File: pages/linux/caffeine.md (hunk 1 above)
Comment (franzqat) [reply pages/linux/caffeine.md:12]: Sure, didn't know that!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**